sos/report/plugins/migration_results.py

```diff
@@ -0,0 +1,21 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin
+
+
+class MigrationResults(Plugin, RedHatPlugin):
+
+    short_desc = 'Information about conversions and upgrades'
```

Comment:
This needs to be in the

    """Information about conversions and upgrades"""

comment style - compare it with other plugins. `short_desc` works well since `4.0`, until then you get:

    # sosreport -l

    ..
    The following plugins are currently disabled:

    ..
     memcached            inactive       memcached distributed memory caching system
     migration_results    inactive       This is the base class for sosreport plugins. Plugins should subclass
        this and set the class variables where applicable.

        plugin_name is a string returned by plugin.name(). If this is set to None
        (the default) class\_.__name__.tolower() will be returned. Be sure to set
        ..
     mongodb              inactive       MongoDB document database